### ISSUE TITLE
Calling a method twice crashes the parser

### ISSUE THREAD

**Issue by netux** [ItsDeltin/Overwatch-Script-To-Workshop#37]
On the latest release (v0.3.12), no matter from where, if a method is called more than once the parser crashes

### Example code
```
method test() {
	return true;
}

rule: "Calling test()" {
	test();
	test();
}
```

**Stacktrace:**
```
[:] Overwatch Script To Workshop v0.3.12
[:] Internal exception.
[:] System.NullReferenceException: Object reference not set to an instance of an object.
   at Deltin.Deltinteger.Parse.Var.get_Range() in C:\Users\HDdel\Documents\GitHub\Overwatch-Workshop-Text\Deltinteger\Deltinteger\Parse\Var.cs:line 140
   at Deltin.Deltinteger.Parse.ScopeGroup.In(IScopeable var) in C:\Users\HDdel\Documents\GitHub\Overwatch-Workshop-Text\Deltinteger\Deltinteger\Parse\Scope.cs:line 43
   at Deltin.Deltinteger.Parse.VarCollection.AssignVar(ScopeGroup scope, String name, Boolean isGlobal, Node node) in C:\Users\HDdel\Documents\GitHub\Overwatch-Workshop-Text\Deltinteger\Deltinteger\Parse\Var.cs:line 91
   at Deltin.Deltinteger.Parse.TranslateRule.ParseUserMethod(ScopeGroup scope, MethodNode methodNode, UserMethod method, IWorkshopTree[] parameters) in C:\Users\HDdel\Documents\GitHub\Overwatch-Workshop-Text\Deltinteger\Deltinteger\Parse\TranslateRule.cs:line 473
   at Deltin.Deltinteger.Parse.TranslateRule.ParseMethod(ScopeGroup scope, MethodNode methodNode, Boolean needsToBeValue) in C:\Users\HDdel\Documents\GitHub\Overwatch-Workshop-Text\Deltinteger\Deltinteger\Parse\TranslateRule.cs:line 440
   at Deltin.Deltinteger.Parse.TranslateRule.ParseStatement(ScopeGroup scope, Node statement, IndexedVar returnVar) in C:\Users\HDdel\Documents\GitHub\Overwatch-Workshop-Text\Deltinteger\Deltinteger\Parse\TranslateRule.cs:line 619
   at Deltin.Deltinteger.Parse.TranslateRule.ParseBlock(ScopeGroup scopeGroup, BlockNode blockNode, Boolean fulfillReturns, IndexedVar returnVar) in C:\Users\HDdel\Documents\GitHub\Overwatch-Workshop-Text\Deltinteger\Deltinteger\Parse\TranslateRule.cs:line 607
   at Deltin.Deltinteger.Parse.TranslateRule..ctor(RuleNode ruleNode, ScopeGroup root, ParsingData parserData) in C:\Users\HDdel\Documents\GitHub\Overwatch-Workshop-Text\Deltinteger\Deltinteger\Parse\TranslateRule.cs:line 37
   at Deltin.Deltinteger.Parse.ParsingData..ctor(String document) in C:\Users\HDdel\Documents\GitHub\Overwatch-Workshop-Text\Deltinteger\Deltinteger\Parse\Parse.cs:line 106
   at Deltin.Deltinteger.Program.Script(String parseFile) in C:\Users\HDdel\Documents\GitHub\Overwatch-Workshop-Text\Deltinteger\Deltinteger\Program.cs:line 74
   at Deltin.Deltinteger.Program.Main(String[] args) in C:\Users\HDdel\Documents\GitHub\Overwatch-Workshop-Text\Deltinteger\Deltinteger\Program.cs:line 50
[:] Done. Press enter to exit.
```

**Comment by ItsDeltin** [comment]
Calling any [multi-action custom method](https://github.com/ItsDeltin/Overwatch-Script-To-Workshop#custom-methods) twice will also crash due to the same bug. 

This will be fixed in the next release.

**Comment by ItsDeltin** [comment]
The crash won't happen if the methods are not in the same scope:
```
if (true) test();
if (true) test();
```

Unfortunately that makes an unnecessarily complicated workshop output,

**Comment by ItsDeltin** [comment]
Fixed in [v0.3.13](https://github.com/ItsDeltin/Overwatch-Script-To-Workshop/releases/tag/v0.3.13).
